### PR TITLE
feat(application-templates-party-application): Email per constituency

### PR DESCRIPTION
**Pull request by stefanhar**
# Email per constituency

Currently we can only send party application submissions to one email.

## What

- Added multiple admin emails
- Fixed admin secret names

## Why

- Allows us to have one email address per constituency

## Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/application-system/api/infra/application-system-api.ts [hunk 1]
```diff
@@ -154,20 +154,34 @@ export const serviceSetup = (services: {
...
-        '/k8s/application-system-form/PARTY_APPLICATION_NORTH_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_NORTH_ASSIGNED_ADMINS',
       PARTY_APPLICATION_SOUTH_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_APPLICATION_SOUTH_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_APPLICATION_SOUTH_ASSIGNED_ADMINS',
       PARTY_LETTER_ASSIGNED_ADMINS:
-        '/k8s/application-system-form/PARTY_LETTER_ASSIGNED_ADMINS',
+        '/k8s/application-system/api/PARTY_LETTER_ASSIGNED_ADMINS',
+
+      PARTY_APPLICATION_RVK_SOUTH_ADMIN_EMAIL:
```
Comment (pshomov) [inline apps/application-system/api/infra/application-system-api.ts:173]: how often are these gonna change 😅?

**Review by pshomov — APPROVED** [review]
lgtm

File: apps/application-system/api/infra/application-system-api.ts (hunk 1 above)
Comment (stefanhar) [reply apps/application-system/api/infra/application-system-api.ts:173]: Once when we insert the correct values then every 4 years.
This is a temporary solution some of these variables will be replaced with the delegation system.
Others will be moved to a admin system that should be part of the next iteration of the system

**Review by albinagu — APPROVED** [review]
sgtm

File: libs/application/template-api-modules/src/lib/modules/templates/party-application/party-application.service.ts [hunk 2]
```diff
@@ -34,6 +41,10 @@ type EndorsementListResponse =
     }
   | ErrorResponse
 
+export type PartyApplicationServiceOptions = {
```
Comment (jeremybarbet) [inline libs/application/template-api-modules/src/lib/modules/templates/party-application/party-application.service.ts:44]: An interface would be better to describe this type. In general `interface` have better perfs than `type`

Comment (stefanhar) [reply libs/application/template-api-modules/src/lib/modules/templates/party-application/party-application.service.ts]: Fixed

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Pull request merged**